lib/Migration/Version000700Date20230916000000.php

```diff
@@ -38,6 +38,10 @@ class Version000700Date20230916000000 extends SimpleMigrationStep {
 			'name' => 'selection',
 			'db_type' => Types::TEXT,
 		],
+		[
```

Comment:
This needs to happen in a separate migration as well, otherwise the usergroup table is not created

It is probably the best to copy the logic of this migration to a new one to create it. if it does not exist.